Content.Client/GameObjects/Components/Power/ApcBoundUserInterface.cs

```diff
@@ -21,12 +23,47 @@ protected override void Open()
         {
             base.Open();
 
-            _window = new ApcWindow(IoCManager.Resolve<IDisplayManager>());
+            _window = new SS14Window(IoCManager.Resolve<IDisplayManager>())
```

Comment:
You should move the logic of creating all the controls to `ApcWindow` and start using it again.

Reply:
Will do

Reply:
Done in the latest commit.